digits/extensions/view/boundingBox/view.py

```diff
@@ -125,10 +129,9 @@ def process_data(
                 box = ((output[0], output[1]), (output[2], output[3]))
                 bboxes.append(box)
                 self.bbox_count += 1
-        digits.utils.image.add_bboxes_to_image(
```

Comment:
I left it there in case there was a use for it in the REST API at some point. I'll just pull it.